packages/skill-lesson/lib/module-progress.ts

```diff
@@ -41,7 +48,7 @@ export async function getModuleProgress({
     'lessonId',
   )
 
-  const moduleProgressLessons = allModuleLessons.map((lesson: Lesson) => {
+  const moduleProgressLessons = allModuleLessons.map((lesson) => {
```

Comment:
able to remove all the type assertions throughout this function now that we are getting typed data from the `getModuleStructure` function